outport/outport.go

```diff
@@ -46,7 +51,32 @@ func (o *outport) SaveBlock(args *indexer.ArgsSaveBlockData) {
 	}
 }
 
+func (o *outport) monitorCompletionOnDriver(function string, driver Driver) chan struct{} {
+	o.logHandler(logger.LogDebug, "outport.monitorCompletionOnDriver starting",
+		"function", function, "driver", driverString(driver))
+	ch := make(chan struct{})
+	go func() {
+		timer := time.NewTimer(o.timeForDriverCall)
+
+		select {
+		case <-ch:
+			o.logHandler(logger.LogDebug, "outport.monitorCompletionOnDriver ended",
```

Comment:
might add an unit test (or extend an existing one) for this print as well

Reply:
added new unit test